pack.ps1

```diff
@@ -112,18 +112,56 @@ function buildNuGetMetaPackageSpec($pkgName)
     $pkgSpecDocument.Save($pkgSpecFilePath)
 }
 
+function isEmptyTranslation($culture)
+{
+    $currentTranslationsFolder = [IO.Path]::Combine($localizationFolderName, $culture)
+    $defaultTranslations = Get-ChildItem $localizationFolderName -File   
+    $currentTranslations = Get-ChildItem $currentTranslationsFolder -File
+    $nonTranslatedContents = 0
+    $contentDiff = 14;
+    $minNonTranslatedContents = 61
```

Comment:
I can try to eliminate some of the crazy numbers, if we can ensure that all `.po` files are well formatted. Have a look to `OrchardCore.Users.Core.po` in `bg-BG` culture for instance

Reply:
/cc @agriffard